server/src/api/modules/groups/services/AddMembersService.ts

```diff
@@ -36,51 +38,69 @@ async function addMembers(payload: AddMembersService): Promise<{ count: number }
     );
   }
 
-  // Find all existing members' ids
-  const existingIds = (
-    await prisma.membership.findMany({
-      where: { groupId: params.id },
-      select: { playerId: true }
+  const count = await prisma
+    .$transaction(async transaction => {
```

Comment:
same thing here, not everything needs to be inside the transaction, only the "mutation" operations